devices.js

```diff
@@ -16465,6 +16465,31 @@ const devices = [
         description: 'Wireless Controlled LED retrofit downlight',
         extend: preset.light_onoff_brightness_colortemp({colorTempRange: [200, 370]}),
     },
+
+    // Aldi
+    {
+        fingerprint: [{modelID: 'TS0505B', manufacturerName: '_TZ3000_j0gtlepx'}],
+        model: '4061458232449',
+        vendor: 'Aldi',
+        description: 'LIGHTWAY Smart Home LED-Leuchtmittel - Reflektor',
```

Comment:
Can you translate the descriptions to English? Besides that no comments.

Reply:
Done, thanks!

Reply:
I have changed the model name of all bulbs because I used the wrong "number" 
And I have also added the missing E14 bulb because I found the information in this issue: https://github.com/Koenkk/zigbee2mqtt/issues/6340

Reply:
Regarding the model, I think you are right. 
I have changed my PR and added also the E14 bulb. 
Maybe this manual is also helpful https://api.eures-gmbh.de/storage/megos_anleitung_de/F122SB62H22A4.5W.pdf